### PR TITLE
Rfpoulos patch 1

### PR DESCRIPTION
**Pull request by rfpoulos**
@rcackerley @smith-taher This is a readme on how the database is set up and how to format API requests.  If this is good/accurate, I propose every time we change either anything in the DB or in DB requests, we update the documentation.

### CODE REVIEW

**Comment by rfpoulos** [comment]
As a tip, when in the "file changes" tab, click "view" in the upper right to get the formatting.

File: README.md [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+
+<h3>Users:</h3>
+    <p class="indent">username character varying(200) UNIQUE NOT NULL,<br>
+    password character varying(200) UNIQUE NOT NULL,<br>
+    userid serial PRIMARY KEY,<br>
+    leaning integer,<br>
+    email character varying(200) UNIQUE NOT NULL</p>
+<h3>Articles:</h3>
+    <p>articleid serial PRIMARY KEY,<br>
+    topic character varying(200) NOT NULL,<br>
```
Comment (smith-taher) [inline README.md]: In looking at the information that is being pulled in from the API, topic is not a referenced object. "Title" is the keyword that is used. We can keep "topic" but change it from "not null".

File: README.md [hunk 2]
```diff
@@ -31,6 +31,10 @@ We are using a Postgres database that have three tables set up as follow:
     
 <h2>Restful API format</h2>
 <h3>GET</h3>
+<h4>Full tables</h4>
+<p>To retrieve JSON of entire database tables, make GET request localhost:3000/{insert_table}.  Optional tables are <em>users</em>, <em>articles</em>, and <em>ratings</em>.</p>
+<h4>Individual entries</h4>
+<p>For individual entries in each table, got to localhost:3000/{insert_table}/{insert_id#}.  For example localhost:3000/users/6 will return the entire column of the user with the serial id 6.  Same for articles and ratings.
 <h3>POST</h3>
```
Comment (smith-taher) [inline README.md]: Great job! You got a lot accomplished yesterday. Is this working now or after I sync with master again?

File: README.md (hunk 1 above)
Comment (rfpoulos) [reply README.md]: This is what the sql file is (copied directly).  Could you change the SQL file first and then I (or you) can change the documentation to match?  I did a function yesterday that the server doesn't need to change when the database is changed, so the SQL file just needs to be formed how you need it.

File: README.md (hunk 2 above)
Comment (rfpoulos) [reply README.md]: This reflects some of the changes on the other pull request too.  Writing the documentation is actually what inspired me to do the other patch so we could post objects when optional values were excluded:-p

File: README.md (hunk 1 above)
Comment (smith-taher) [reply README.md]: I'm happy to make that change.

**Pull request merged**